Backend/src/main/java/ca/mcgill/cooperator/controller/NotificationController.java

```diff
@@ -43,6 +43,20 @@ public NotificationDto getNotificationById(@RequestParam int id) {
 
         return ControllerUtils.convertToDto(n);
     }
+    
+
+    /**
+     * Sets notification seen by id
+     *
+     * @param id
+     * @return NotificationDto
+     */
+    @GetMapping("/seen/{id}")
+    public NotificationDto setNotificationSeen(@RequestParam int id) {
+        Notification n = notificationService.getNotification(id);
+        notificationService.setSeen(n);
```

Comment:
Maybe change `setSeen` to `markAsRead` so that it's more obvious what that method is doing?